doc/lua_api.md

```diff
@@ -4007,8 +4007,9 @@ Helper functions
     * X1, Y1, ... Z2 are coordinates
     * `relative_to`: Optional. If set to a position, each coordinate
       can use the tilde notation for relative positions
-    * Tilde notation: "~": Relative coordinate
-                      "~<number>": Relative coordinate plus <number>
+    * Tilde notation
+      * "\~": Relative coordinate
+      * "\~<number>": Relative coordinate plus \<number\>
```

Comment:
This is misleading if you read it in text form, it looks like the backslash is part of the syntax.
How about:
```suggestion
      * `"~"`: Relative coordinate
      * `"~<number>"`: Relative coordinate plus \<number\>
```